.github/workflows/release.yml

```diff
@@ -3,6 +3,10 @@ on:
   push:
     branches:
       - master
+      
+  pull_request:
+    paths:
+      - '.github/workflows/release.yml'
```

Comment:
This is temporary while testing, right?

Reply:
Yep! We can remove it now that it's working.